verbose_debugging_reports.md

```diff
@@ -288,6 +295,7 @@ The trigger was rejected due to an internal error.
 [event-level report body]: https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#attribution-reports
 [filter data]: https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#optional-attribution-filters
 [insufficient budget]: https://github.com/WICG/attribution-reporting-api/blob/main/AGGREGATE.md#contribution-bounding-and-budgeting
+[insufficient named budget]: https://wicg.github.io/attribution-reporting-api/#attribution-source-named-budgets
```

Comment:
It might be better to link to the explainer description of this instead of a relatively non-descriptive struct field in the spec.

Reply:
Done. Nice catch!